SU2_CFD/src/integration_structure.cpp

```diff
@@ -154,9 +154,9 @@ void CIntegration::Space_Integration(CGeometry *geometry,
       case DISPLACEMENT_BOUNDARY:
         solver_container[MainSolver]->BC_Normal_Displacement(geometry, solver_container, numerics[CONV_BOUND_TERM], config, iMarker);
         break;
-      case FLOWLOAD_BOUNDARY:
-        solver_container[MainSolver]->BC_Flow_Load(geometry, solver_container, numerics[CONV_BOUND_TERM], config, iMarker);
-        break;
+//      case FLOWLOAD_BOUNDARY:
```

Comment:
Can we remove these instead of commenting them out? There are a few other places where lines have been commented that might be removed instead.